index.html

```diff
@@ -68,6 +68,177 @@ <h1 class="main__title">I'm Alexandra. <br> Glad to see you!</h1>
     </main>
     <!-- END -->
 
+    <!-- WORKS SECTION -->
+    <section id="portfolio" class="work__section">
+        <article class="work__description">
+            <figure class="work__image__container">
+                <img src="/assets/work1.png" alt="first work image" class="work__image">
```

Comment:
Please as the previous reviewer requested change the images `src` attributes. Please consider all the requirements and update accordingly. https://github.com/Feliperto/PortfolioWebsite-Microverse/pull/2#discussion_r1213719175

Reply:
Hello @

Reply:
Hello @shahadat3669 I've solved it before, i don't know what change do you want me to do it. Can you tell me please. I have the image with the name work1 and the src="/assets/work1.png". Thank you

Reply:
Hi @Feliperto, check [it](https://github.com/Feliperto/PortfolioWebsite-Microverse/pull/2#discussion_r1213719175). As the previous reviewer asked you to add a dot(.) before "/" symbol. 
![image](https://github.com/Feliperto/PortfolioWebsite-Microverse/assets/55840999/a60686ce-7c99-499a-ab8d-4aa9d796849f)

Reply:
Very thank for answer, I'll change right now